test-vectors/presentation_exchange/select_credentials_go.json

```diff
@@ -0,0 +1,270 @@
+          "eyJ0eXAiOiJKV1QiLCJhbGciOiJFZERTQSIsImtpZCI6ImRpZDprZXk6ejZNa2ZIQk5tYzRjQ2hOdER2YXIxOFRaaWdmWHQ3UDY1RkJHd3FRVHR4UW9RUG5HI3o2TWtmSEJObWM0Y0NoTnREdmFyMThUWmlnZlh0N1A2NUZCR3dxUVR0eFFvUVBuRyJ9.eyJpc3MiOiJkaWQ6a2V5Ono2TWtmSEJObWM0Y0NoTnREdmFyMThUWmlnZlh0N1A2NUZCR3dxUVR0eFFvUVBuRyIsInN1YiI6ImRpZDprZXk6ejZNa2ZIQk5tYzRjQ2hOdER2YXIxOFRaaWdmWHQ3UDY1RkJHd3FRVHR4UW9RUG5HIiwidmMiOnsiQGNvbnRleHQiOlsiaHR0cHM6Ly93d3cudzMub3JnLzIwMTgvY3JlZGVudGlhbHMvdjEiXSwidHlwZSI6WyJWZXJpZmlhYmxlQ3JlZGVudGlhbCIsIlN0cmVldENyZWQiXSwiaWQiOiJ1cm46dXVpZDoxM2Q1YTg3YS1kY2Y1LTRmYjktOWUyOS0wZTYyZTI0YzQ0ODYiLCJpc3N1ZXIiOiJkaWQ6a2V5Ono2TWtmSEJObWM0Y0NoTnREdmFyMThUWmlnZlh0N1A2NUZCR3dxUVR0eFFvUVBuRyIsImlzc3VhbmNlRGF0ZSI6IjIwMjMtMTItMDdUMTc6MTk6MTNaIiwiY3JlZGVudGlhbFN1YmplY3QiOnsiaWQiOiJkaWQ6a2V5Ono2TWtmSEJObWM0Y0NoTnREdmFyMThUWmlnZlh0N1A2NUZCR3dxUVR0eFFvUVBuRyIsIm90aGVydGhpbmciOiJvdGhlcnN0dWZmIn19fQ.FVvL3z8LHJXm7lGX2bGFvH_U-bTyoheRbLzE7zIk_P1BKwRYeW4sbYNzsovFX59twXrnpF-hHkqVVsejSljxDw"
+        ]
+      },
+      "output": {
+        "selectedCredentials": [
+          "eyJhbGciOiJFZERTQSIsImtpZCI6ImRpZDpqd2s6ZXlKcmRIa2lPaUpQUzFBaUxDSmpjbllpT2lKRlpESTFOVEU1SWl3aWVDSTZJa013VlhrNGN6SnVla1JhTjFsdmRXeFFUM010YjBabFEwWkxaVzAzY1hST1FWVTNiVzB0TjNaaFkxRWlmUSMwIiwidHlwIjoiSldUIn0.eyJpc3MiOiJkaWQ6andrOmV5SnJkSGtpT2lKUFMxQWlMQ0pqY25ZaU9pSkZaREkxTlRFNUlpd2llQ0k2SWtNd1ZYazRjekp1ZWtSYU4xbHZkV3hRVDNNdGIwWmxRMFpMWlcwM2NYUk9RVlUzYlcwdE4zWmhZMUVpZlEiLCJqdGkiOiJ1cm46dmM6dXVpZDo3MjllMTc2ZS1mYjNlLTQyOTktOWI3Yi02MGQzNmVkMzQxNmUiLCJuYmYiOjE3MTQxNjE1NjEsInN1YiI6ImRpZDpqd2s6ZXlKcmRIa2lPaUpQUzFBaUxDSmpjbllpT2lKRlpESTFOVEU1SWl3aWVDSTZJa013VlhrNGN6SnVla1JhTjFsdmRXeFFUM010YjBabFEwWkxaVzAzY1hST1FWVTNiVzB0TjNaaFkxRWlmUSIsInZjIjp7IkBjb250ZXh0IjpbImh0dHBzOi8vd3d3LnczLm9yZy8yMDE4L2NyZWRlbnRpYWxzL3YxIl0sInR5cGUiOlsiVmVyaWZpYWJsZUNyZWRlbnRpYWwiLCJTdHJlZXRDcmVkZW50aWFsIl0sImlzc3VlciI6ImRpZDpqd2s6ZXlKcmRIa2lPaUpQUzFBaUxDSmpjbllpT2lKRlpESTFOVEU1SWl3aWVDSTZJa013VlhrNGN6SnVla1JhTjFsdmRXeFFUM010YjBabFEwWkxaVzAzY1hST1FWVTNiVzB0TjNaaFkxRWlmUSIsImNyZWRlbnRpYWxTdWJqZWN0Ijp7ImlkIjoiZGlkOmp3azpleUpyZEhraU9pSlBTMUFpTENKamNuWWlPaUpGWkRJMU5URTVJaXdpZUNJNklrTXdWWGs0Y3pKdWVrUmFOMWx2ZFd4UVQzTXRiMFpsUTBaTFpXMDNjWFJPUVZVM2JXMHROM1poWTFFaWZRIiwibmFtZSI6IlNhdG9zaGkgVGFjb21vdG8ifSwiaWQiOiJ1cm46dmM6dXVpZDo3MjllMTc2ZS1mYjNlLTQyOTktOWI3Yi02MGQzNmVkMzQxNmUiLCJpc3N1YW5jZURhdGUiOiIyMDI0LTA0LTI2VDE5OjU5OjIxWiJ9fQ.MoPYXkSASXEgySIc59HnSN8576cu5q8QC5tCG3PKr3j-glvZNa12j_P563FUVzx7PeFD3QkJne1RYBDOj3OcBw"
```

Comment:
this output is also correct since it is the only VC that includes the type `StreetCredential` at location `$.vc.type`